frameworks/C/h2o/setup.sh

```diff
@@ -5,7 +5,7 @@ fw_depends postgresql h2o mustache-c yajl
 H2O_APP_HOME="${IROOT}/h2o_app"
 BUILD_DIR="${H2O_APP_HOME}_build"
 H2O_APP_PROFILE_PORT="54321"
-H2O_APP_PROFILE_URL="http://127.0.0.1:$H2O_APP_PROFILE_PORT"
+H2O_APP_PROFILE_URL="http://TFB-database:$H2O_APP_PROFILE_PORT"
```

Comment:
This is wrong: The URL has nothing to do with the database (note that it is used as a parameter for curl); it is just the location at which the HTTP server is temporarily accessible while gathering the profiling data.